### PR TITLE
Issue 42 tools resources

### PR DESCRIPTION
**Pull request by laurenchamps**
Create Tools & Resources page and add content from README.

Add link in navigation on all other existing pages plus template.html.

### CODE REVIEW

File: sub-page/accessibility-course.html
Comment (sarahesbie) [inline sub-page/accessibility-course.html:1]: @laurenchamps can you change the link from 
`<a href="/tools-and-resources.html">`
to
`<a href="tools-and-resources.html">`

for some reason github pages complains about the slash.

File: sub-page/web-developer-pledge.html
Comment (sarahesbie) [inline sub-page/web-developer-pledge.html:1]: @laurenchamps can you change the link from
`<a href="/tools-and-resources.html">`
to
`<a href="tools-and-resources.html">`

for some reason github pages complains about the slash.

**Review by sarahesbie — CHANGES_REQUESTED** [review]
Heya @laurenchamps just a couple of super quick edits before we merge this one :)

File: sub-page/web-developer-pledge.html
Comment (laurenchamps) [reply sub-page/web-developer-pledge.html:1]: Hey Sarah, I've removed the slash on both these pages. Should be good now I hope!

**Review by sarahesbie — APPROVED** [review]

**Pull request merged**